### DISCUSSION TITLE
Trail Length Exceeding 250 Frames

### DISCUSSION THREAD

**Discussion by kjagasia** [talmolab/sleap#1769] [Ideas]
Is there any way to display trail lengths that are longer than the previous 250 frames? Specifically, for my video with 451 frames, how can I display the trails for the entire video?

**Comment by roomrys** [comment]
Hi @kjagasia,

Unfortunately, there is no option at the moment to display a trail length other than those provided in the options list.

I was going to suggest a workaround of manually editing the `trail length` attribute in your `preferences.yaml` that is saved inside the `.sleap` folder in your home directory, but it looks like we aren't using (or saving) the `trail length` attribute, so that won't work either.

I incorrectly assumed `trail length` preference would be used to initialize the `TrackTrailOverlay` class, but alas, it is not:
https://github.com/talmolab/sleap/blob/834c68a11842b843d3c191041f5b2390eae307ce/sleap/gui/app.py#L1024-L1028

This would be a good idea for a new feature though!

Sorry about that,
Liezl
